packages/server/src/error.rs

```diff
@@ -31,6 +30,22 @@ impl From<bson::oid::Error> for AppError {
     }
 }
 
+impl std::fmt::Display for AppError {
```

Comment:
Why do you implement Display trait instead of using the existing one?
It doesn't seem related to the change of the env! macro.

Reply:
You're right, it isn't related, I got rid of the `#derive` as a part of the "Also removed some unused dependencies" changes, we brought a whole crate for a deriving single trait for a single struct, which could just be simply implemented in less than 10 lines. This is one of those cases where I think bringing a crate isn't necessary, for similar reasons we discussed some time ago about the builder crates.